R/map.R

```diff
@@ -182,3 +182,41 @@ map_if <- function(.x, .p, .f, ...) {
   .x[sel] <- lapply(.x[sel], .f, ...)
   .x
 }
+
+#' Modify elements of a specified subset.
+#'
+#' @inheritParams map
+#' @param .at A character vector of names or a numeric vector of
```

Comment:
I didn't document it because this functionality overlaps with `map_if()` and I was afraid it would confuse things. Should I document it anyway? Or maybe make `map_at()` fail with logical vectors?